crates/batch-prover/src/runner.rs

```diff
@@ -354,6 +354,7 @@ where
                                 // This block failed to process, add remaining L2 blocks to queue including this one.
                                 let remaining_l2s = l2_blocks[index..].to_vec();
                                 pending_l2_blocks.extend(remaining_l2s);
+                                break;
```

Comment:
It will stop syncing in case of error. Is it ok?

Reply:
no that's just the for above